contracts/Conference.sol

```diff
@@ -179,16 +176,16 @@ contract Conference is GroupAdmin {
         if (!isRegistered(_addr)) {
             return false;
         }
-
         // check the attendance maps
-        if (0 < attendanceMaps.length) {
+        else if (0 < attendanceMaps.length) {
             Participant storage p = participants[_addr];
             uint pIndex = p.participantIndex - 1;
             uint map = attendanceMaps[uint(pIndex / 256)];
             return (0 < (map & (2 ** (pIndex % 256))));
         }
-
-        return participants[_addr].attended;
+        else {
```

Comment:
Is this the case when `finalize` has not happened? Is it more explict to say `if(!ended) {return false}` ? If people reaches this stage even after the check, then it must be a bug to throw error.

Reply:
I think it's fine as it is. As a fallback it will always return 0.